helm/chart/maesh/charts/metrics/values.yaml

```
@@ -18,6 +27,15 @@ grafana:
   #     cpu: "100m"
 
 prometheus:
+  service:
+    type: ClusterIP
+    port: 9090
+
+    ## Specify the nodePort value for the NodePort service types.
+    ## ref: https://kubernetes.io/docs/concepts/services-networking/service/#nodeport
+    ##
+    # nodePort: null
```

I think that it's better to use a dummy `nodePort` for the example instead of null.

```suggestion
    # nodePort: 30090
```